### ISSUE TITLE
Don't shelve md5 and google provided METS-like xml file.

### ISSUE THREAD

**Issue by jcoyne** [sul-dlss/repository-api#48]

**Comment by jcoyne** [comment]
@andrewjbtw @peetucket do we want to deposit the md5 and METS-like xml file or just mark it not to be shelved?

**Comment by peetucket** [comment]
Yes, I think the idea was to mark them as preserve only (so they can deposited but not shelved).  Is that right Andrew?

**Comment by andrewjbtw** [comment]
I thought we were going to shelve and preserve the XML file because it has information that could be useful to people processing the books, like the section that indicates which pages have images or not.

**Comment by jcoyne** [comment]
Got it, thanks.

**Comment by peetucket** [comment]
andrew, should we update the title of the ticket to reflect this then?

**Comment by andrewjbtw** [comment]
I think so.
